spring-cloud-contract-starters/spring-cloud-starter-contract-stub-runner/pom.xml

```diff
@@ -33,6 +33,35 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-contract-wiremock</artifactId>
 		</dependency>
+		<!-- for Stub Runner's Aether -->
```

Comment:
I'm adding these even though I shouldn't since the dependencies should be shaded... I'll check it out again